flow/Histogram.h

```diff
@@ -58,16 +59,32 @@ HistogramRegistry& GetHistogramRegistry();
  */
 class Histogram final : public ReferenceCounted<Histogram> {
 public:
-	enum class Unit { microseconds, bytes, bytes_per_second };
+	enum class Unit { microseconds, bytes, bytes_per_second, percentage, record_counter };
 
 private:
-	static const std::unordered_map<Unit, std::string> UnitToStringMapper;
-
-	Histogram(std::string const& group, std::string const& op, Unit unit, HistogramRegistry& registry)
-	  : group(group), op(op), unit(unit), registry(registry), ReferenceCounted<Histogram>() {
+	// static const std::unordered_map<Unit, std::string> UnitToStringMapper;
```

Comment:
It is not clear to me why this map or the `Unit` enumeration need to exist.  Is there a reason each Histogram just store a string of its unit name?  Is there some code that has to know what the units are, qualitatively, to do the right thing?

If the Unit enumeration needs to exist, and the map only serves to get string equivalents, this would be better done as a static const array of const string literals defined on the line immediately after the `Unit` enum.